pkg/rid/store/cockroach/garbage_collector.go

```diff
@@ -0,0 +1,64 @@
+package cockroach
+
+import (
+	"context"
+
+	"github.com/interuss/dss/pkg/logging"
+	"github.com/interuss/dss/pkg/rid/repos"
+	"go.uber.org/zap"
+)
+
+type GarbageCollector struct {
+	repos  repos.Repository
+	writer string
+}
+
+func NewGarbageCollector(repos repos.Repository, writer string) *GarbageCollector {
+	return &GarbageCollector{
+		repos:  repos,
+		writer: writer,
+	}
+}
+
+func (gc *GarbageCollector) DeleteExpiredRecords(ctx context.Context) {
+	gc.DeleteExpiredISAs(ctx)
+	gc.DeleteExpiredSubscriptions(ctx)
+}
+
+func (gc *GarbageCollector) DeleteExpiredISAs(ctx context.Context) {
+	logger := logging.WithValuesFromContext(ctx, logging.Logger)
+	expiredISAs, err := gc.repos.ListExpiredISAs(ctx, gc.writer)
+	if err != nil {
+		logger.Panic("Failed to list expired ISAs", zap.Error(err))
```

Comment:
Lets use stacktrace.propagate as it provides better logging information. GC is a necessary flow but should not result in Panic